### ISSUE TITLE
A11y_VisualStudioCodeClient__Editor_Usable: Expecting screen reader to announce when we undo the code.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#196818]
### GitHubTags:
#A11ySev4;#A11yTCS;#A11yUsability;#DesktopApp;#Visual Studio Code Client;#DesktopApp;#BM-VisualStudioCodeClient-Win32-Aug2022;#NVDA;#Win32;#Win11;#A11yeDAD;#AINotInScope;

### Environment Details:​
Application Name: Visual Studio Code
Version: 1.83.1(user setup)
V8: 11.4.183.29-electron.0
OS build 25967.1010
NVDA- version 2023.1(2023.1.0.27913)

### Repro Steps:
Turn on NVDA
Launch Visual Studio Code
Navigate to any line and press ‘ctrl+ z’.
Verify whether screen reader to announce when we undo the code.

### Actual:
Expecting screen reader to announce when we undo the code.

### Expected:
NVDA to announce that the selected content is erased/undo.

### User Impact:
Screen reader users miss the information if the functionality is performed or not.

### Attachment
A11y_VisualStudioCodeClient__Editor_Usable

**Comment by meganrogge** [comment]
Since one's cursor is in the area where the undo occurred, I think it can be verified that way? Maybe we could use an audio cue or an alert here?

Thoughts @jooyoungseo or @rperez030 ?

**Comment by jooyoungseo** [comment]
FYI, when pressing ctrl+z, the undo content is announced to screen readers. Tested with NVDA and JAWS. IMO, announcing additional prefix (e.g., undo) is not necessary because this is triggered via user explicit gesture. More precisely, this should be supported via screen reader itself. For example, copy, cut, paste, delete, and undo are standard editing events and their labels are already togglable in some screen readers and outside the scope of VSCode itself.

**Comment by meganrogge** [comment]
@kupatkar99 perhaps some screen reader setting is causing it to not work for you?

**Comment by kupatkar99** [comment]
As per the discussion with SME copy, cut, paste, delete, and undo are standard editing events and their labels are already togglable in some screen readers. Hence Closing the bug. #Closed;
